packages/o/opencv/xmake.lua

```diff
@@ -97,6 +99,12 @@ package("opencv")
             local arch = (package:is_arch("x86_64") and "x64" or "x86")
             local linkdir = (package:config("shared") and "lib" or "staticlib")
             package:add("linkdirs", path.join(arch, "mingw", linkdir))
+        elseif package:is_plat("android") then
+            local arch = package:targetarch()
+            local linkdir = (package:config("shared") and "lib" or "staticlibs")
```

Comment:
`sdk\native\libs\arm64-v8a\libopencv_signal.so`
So this should be
```suggestion
            local linkdir = (package:config("shared") and "libs" or "staticlibs")
```
Though CI is probably going to work since it static only, ```sdk/native/staticlibs/arm64-v8a/libopencv_bgsegm.a```